backend/app/services/auth_service.py

```
@@ -0,0 +1,37 @@
+from fastapi import HTTPException, status, Depends
+from fastapi.security import OAuth2PasswordBearer
+from supabase import Client, create_client
```

Flake8 (7.2.0)

[error] 'supabase.Client' imported but unused

(F401)